cern_access/indico_cern_access/templates/identity_data_form.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'events/display/conference/base.html' %}
+
+{% from 'forms/_form.html' import simple_form %}
+
+{% block title %}
+    {% trans %}Additional data form{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    {% if access_request is none or not access_request.is_accepted %}
+        <div class="info-message-box">
+            <div class="message-box-content">
+                <span class="icon"></span>
+                <div class="message-text">
+                    {% trans %}
+                        You have not been granted access to the CERN site yet.
+                    {% endtrans %}
+                </div>
+            </div>
+        </div>
+    {% elif not access_request.has_identity_info %}
+        {{ simple_form(form, back_button=False) }}
+    {% else %}
+        <div class="success-message-box">
+            <div class="message-box-content">
+                <span class="icon"></span>
+                <div class="message-text">
+                    {% trans %}
```

Comment:
`{% trans -%}...{%- endtrans %}` (you could also strip the surrounding whitespace on the outside if you want to)